### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by krisl**
thanks for the great tutorial!

### CODE REVIEW

File: docs/first-step/run-executable.rst [hunk 1]
```diff
@@ -4,9 +4,9 @@
 Build and run executable
 ------------------------
 
-In this section we will build and run ``foo`` executable. In case if IDE was
-used you can do it by opening project in IDE or using command line (it doesn't
-matter how project was generated before: using GUI or CLI version of CMake).
+In this section we will build and run the ``foo`` executable. You can do
+it by opening the project in an IDE or by using the command line (It doesn't
```
Comment (ruslo) [inline docs/first-step/run-executable.rst:8]: > (It doesn't

Should it be lowercase `(it doesn't` ?

File: docs/first-step/run-executable/cli-xcode.rst [hunk 2]
```diff
@@ -4,15 +4,15 @@
 CLI: Xcode
 ----------
 
-To build Xcode project from command line ``xcodebuild`` can be used.
+To build and Xcode project from the command line, ``xcodebuild`` can be used.
```
Comment (ruslo) [inline docs/first-step/run-executable/cli-xcode.rst:7]: `To build and Xcode` -> `To build an Xcode`

File: docs/first-step/run-executable.rst (hunk 1 above)
Comment (krisl) [reply docs/first-step/run-executable.rst:8]: hmm yeah probably.  ooh theres another "the" to add here, Ill update it.
Perhaps even you want to consider removing the parentheses alltogether?

File: docs/first-step/run-executable/cli-xcode.rst (hunk 2 above)
Comment (krisl) [reply docs/first-step/run-executable/cli-xcode.rst:7]: yup, thanks

**Review by ruslo — APPROVED** [review]
Looks good, thank you 👍

File: docs/first-step/run-executable.rst (hunk 1 above)
Comment (ruslo) [reply docs/first-step/run-executable.rst]: > Perhaps even you want to consider removing the parentheses alltogether?

Up to you, I'm okay with either.

**Pull request merged**